api/src/test/resources/org/openmrs/api/include/PersonServiceTest-extranames.xml

```diff
@@ -11,26 +11,26 @@
-  <person_name person_name_id="3" preferred="true" person_id="3" prefix="" given_name="I" middle_name="am" family_name="voided" family_name2="Johnson" creator="1" date_created="2005-01-01 00:00:00.0" voided="true" uuid="1be83481-5ace-4a97-8510-d2e1257b6f6a"/>
-  <person_name person_name_id="4" preferred="true" person_id="4" prefix="Mr." given_name="John" middle_name="Andrew" family_name="Johnson" family_name2="" creator="1" date_created="2005-01-01 00:00:00.0" voided="false" uuid="23da3d4a-9a12-44b7-bcc0-7ed72664adda"/>
-  <person_name person_name_id="5" preferred="true" person_id="5" prefix="Mrs." given_name="Jean" middle_name="Joan" family_name="Claude" family_name2="Johnson" creator="1" date_created="2005-01-01 00:00:00.0" voided="false" uuid="5e6571cc-c7f2-41de-b289-f55f8fe79c6f"/>
-  <person_name person_name_id="6" preferred="false" person_id="5" prefix="Mrs." given_name="Jean" middle_name="Foobar" family_name="Claude" family_name2="Johnson" creator="1" date_created="2005-01-01 00:00:00.0" voided="false" uuid="5e6571cc-c7f2-41de-b289-f55f8fe79ddd"/>
-  <patient patient_id="2" creator="1" date_created="2005-01-01 00:00:00.0" voided="false"/>
-  <patient patient_id="3" creator="1" date_created="2005-01-01 00:00:00.0" voided="true"/>
-  <patient patient_id="4" creator="1" date_created="2005-01-01 00:00:00.0" voided="false"/>
-  <patient patient_id="5" creator="1" date_created="2005-01-01 00:00:00.0" voided="false"/>
-  <patient_identifier patient_identifier_id="51" patient_id="2" identifier="1234" identifier_type="1" preferred="1" location_id="1" creator="1" date_created="2005-01-01 00:00:00.0" voided="false" uuid="64020a1a-4dc8-417b-b12a-0e91518f68f2"/>
-  <patient_identifier patient_identifier_id="41" patient_id="2" identifier="12345" identifier_type="1" preferred="0" location_id="1" creator="1" date_created="2004-01-01 00:00:00.0" voided="false" uuid="42236d1a-0b5e-43bd-a562-973104f1484e"/>
-  <patient_identifier patient_identifier_id="31" patient_id="3" identifier="4567" identifier_type="1" preferred="1" location_id="1" creator="1" date_created="2005-01-01 00:00:00.0" voided="false" uuid="2d7c3067-346a-4cc8-9f67-66bb5b149153"/>
-  <patient_identifier patient_identifier_id="21" patient_id="4" identifier="5634" identifier_type="1" preferred="1" location_id="1" creator="1" date_created="2005-01-01 00:00:00.0" voided="true" uuid="df6633a5-3896-498f-97bd-3160ffa05d0e"/>
-  <patient_identifier patient_identifier_id="11" patient_id="5" identifier="563422" identifier_type="1" preferred="1" location_id="1" creator="1" date_created="2005-01-01 00:00:00.0" voided="false" uuid="f6e43b2a-e331-496b-87f7-27b64d84a015"/>
+	<person person_id="2" gender="M" dead="false" creator="1" date_created="2005-01-01 00:00:00.0" voided="false" uuid="1716432a-4bf9-436c-baad-f03236e960e1"/>
```

Comment:
I don't particularly love converting the leading white space to tabs here. Is it possible to undo it?